frontend/src/components/interview/interview.tsx

```diff
@@ -0,0 +1,63 @@
+import { DataStatus } from 'common/enums/enums';
+import {
+  FC,
+  InterviewsGetAllItemResponseDto,
+  InterviewsUpdateRequestDto,
+} from 'common/types/types';
+import { Spinner } from 'components/common/common';
+import {
+  useAppDispatch,
+  useAppSelector,
+  useEffect,
+  useParams,
+} from 'hooks/hooks';
+import { interviewActions } from 'store/actions';
+
+import { InterviewItem } from './components/components';
+import styles from './styles.module.scss';
+
+const Interview: FC = () => {
+  const dispatch = useAppDispatch();
+  const { id } = useParams();
+  const { interview, dataStatus, interviewers } = useAppSelector((state) => ({
+    interview: state.interview.interview,
+    dataStatus: state.interview.dataStatus,
+    interviewers: state.interview.interviewers,
+  }));
+
+  useEffect(() => {
+    dispatch(interviewActions.getInterview({ id: Number(id) }));
+  }, []);
+
+  useEffect(() => {
+    if (interview) {
+      dispatch(
+        interviewActions.getInterviewersByCategory({
+          categoryId: interview.courseCategory.id,
+        }),
+      );
+    }
+  }, [interview]);
```

Comment:
I'm not following. Why are we fetching interviews by category inside interview when we got an interview?

Reply:
@PavelSukhinin we are fetching all existing interviewers by the category of the interview, and passing them to the Interview application, where user can swap to edit mode and select another available interviewer from the select